apps/frontend/src/components/cards/controltable/ControlTable.vue

```diff
@@ -133,6 +139,16 @@ export default class ControlTable extends Vue {
     }
   }
 
+ /** Expands all elements within items asynchronously */
+  async expandAllEvaluations(): Promise<void> {
+    this.items.forEach(async (item) => {
+      const i = this.expanded.indexOf(item.key);
+      if (i < 0) {
+        this.expanded.push(item.key);
+      }
+    })
+  }
```

Comment:
Instead of doing a check for every single element if it is already in the array, can we save ourselves the check and just replace the whole `this.expanded` list?

```
this.expanded = this.items.map((items) => items.key);
```